src/nix/meson.options

```diff
@@ -0,0 +1,10 @@
+# vim: filetype=meson
+
+option('state-dir', type : 'string', value : '/nix/var',
```

Comment:
This is defined to a disk location that you are not actually allowed to use?

Reply:
We don't touch this patch at build time, but it is hardcoded into the executable. If Meson just thought of it as a string that would be fine, I think.